source/georef/georef.cpp

```diff
@@ -632,7 +632,7 @@ std::pair<GeoRef::ProjectionByMode, bool> GeoRef::project_stop_point(const type:
         nt::Mode_e::Car // CarNoPark -> Car
     }}};
 
-    for (auto const &mode_layer: mode_to_layer) {
+    for (auto const mode_layer: mode_to_layer) {
```

Comment:
did ya remove that on purpose ? did you had a warning on this ?

Reply:
Yep, clang warns on extra refs and missing refs (it's the same warning : either all of them or none. I chose to keep all).
Here we always get a copy, so no point having a ref.

Reply:
I guess that's a general comment for all the references being removed in this PR